packages/splits-kit/src/components/DisplaySplit/index.tsx

```diff
@@ -38,11 +44,15 @@ const DisplaySplit = ({
     isLoading: isLoadingMetadata,
   } = useSplitMetadata(address)
 
+  const includeActiveBalances = true
+  const erc20TokenList = isAlchemyChainId(chainId)
```

Comment:
you know...this should probably use the same function that the core sdk uses. let me see if i can make that change. then don't need to define this here.